viewer.py

```diff
@@ -103,17 +112,14 @@ def get_db_mtime(self):
 
 def generate_asset_list():
     logging.info('Generating asset-list...')
-
-    now = datetime.utcnow()
-    enabled_assets = [a for a in assets_helper.read(db_conn) if a['is_enabled']]
-    future_dates = [a[k] for a in enabled_assets for k in ['start_date', 'end_date'] if a[k] > now]
-    deadline = sorted(future_dates)[0] if future_dates else None
```

Comment:
Change for: https://github.com/wireload/screenly-ose/commit/5e3828a2f803926fc5cd69ba5dec8b79864ec45b